### DISCUSSION TITLE
Problema al enviar respuestas a RLXMOOC

### DISCUSSION THREAD

**Discussion by cristianpenagos** [rramosp/ai4eng.v1#294] [01 - INTRODUCCIÓN]
Buenos dias.

Tengo el siguiente error al enviar soluciones a la plataforma

---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
[<ipython-input-12-4cdaf622784a>](https://localhost:8080/#) in <cell line: 1>()
----> 1 student.submit_task(globals(), task_id="task_01");

AttributeError: 'NoneType' object has no attribute 'submit_task'

tengo una duplicidad en el registro dado que matricule esta materia en el semestre 2023-2

mi correo: cristian.penagos@udea.edu.co

**Comment by Felipe-RA** [comment]
Hola @cristianpenagos 

Anotado!, aunque la discusión para esto es la #212 !

No te preocupes, ya lo puse allí. En cuanto te lo arreglemos actualizaremos allí mismo!!
